package.json

```diff
@@ -193,9 +193,11 @@
           "padding-line-between-statements": 0,
           "quotes": 0,
           "unicorn/catch-error-name": 0,
+          "unicorn/no-console-spaces": 0,
           "node/no-deprecated-api": 0,
           "@typescript-eslint/member-ordering": "never",
-          "@typescript-eslint/restrict-plus-operands": "never"
+          "@typescript-eslint/restrict-plus-operands": "never",
+          "import/default": 0
```

Comment:
Prevents a default module export being required by every module.

Reply:
We will probably need to remove this later once we are done with migration